spec.bs

```diff
@@ -1523,13 +1523,10 @@ For now, see [https://github.com/wicg/observable#operators](https://github.com/w
        :: [=Reject=] |p| with the passed in <var ignore>error</var>.
 
        : [=internal observer/complete steps=]
-       :: [=Resolve=] |p| with {{undefined}}.
+       :: [=Reject=] |p| with a new {{RangeError}}, and message of "No values in Observable".
```

Comment:
```suggestion
       :: [=Reject=] |p| with a new {{RangeError}}.
```

We don't specify the specific message in errors. See https://streams.spec.whatwg.org/#ts-constructor for example.

Reply:
Would you suggest I also raise a pr into wpt to remove the [test assertions for that message](https://github.com/web-platform-tests/wpt/blob/3585305e8365779fac42dd87e4f52c89c3e884a6/dom/observable/tentative/observable-first.any.js#L52)?

Reply:
Great catch, yeah I think so! That test seems a little too aggressive probably.

Reply:
@domfarolino raised here 👉 https://github.com/web-platform-tests/wpt/pull/48717